index.html

```diff
@@ -3989,10 +3989,24 @@ <h3>JSON-LD</h3>
 
         <p>
 As elaborated upon in Section <a href="#json-processing"></a>, some software
-applications might not perform full JSON-LD processing.
-<a>Conforming document</a> authors are urged to not use JSON-LD keywords that
-affect document processing in the `@context` value, such as globally setting
-`@base` and `@language`, in order to increase interoperability.
+applications might not perform full JSON-LD processing. 
+<a>Conforming document</a> authors are advised that 
+usage of certain JSON-LD features might reduce 
+interoperability. These features include:
```

Comment:
```suggestion
applications might not perform dynamic JSON-LD processing, electing to
use static processing that will, for example, reject documents with unrecognized
contexts. To increase interoperability, <a>conforming document</a> authors are
urged to avoid using JSON-LD features that are easily detected only when
performing dynamic JSON-LD processing. <a>Conforming document</a> authors
are advised that usage of certain JSON-LD features might reduce 
interoperability. These features include:
```